### PR TITLE
Added Manual installation

### LINKED ISSUE #22: Manual Installation
`CocoaPods` and `Carthage` are awesome tools and make our life really easier, but there are some devs who still don't know how to use them. 

It would be cool to add the _Manual installation_ guide in your `README.md`. You can take a look at my [iOS Readme Template](https://github.com/awesome-labs/iOS-readme-template) to see how you can do it.

### PR DESCRIPTION
**Pull request by lfarah**
fixes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ github "MailOnline/Reactor"
 pod 'MOReactor', '~> 0.9'
 ```
 
+#### Manually
+1. Download and drop ```/Reactor``` folder in your project.  
+2. Congratulations!  
+
```
Comment (RuiAAPeres) [inline README.md:58]: The `Congratulation` step  is pointless.

**Comment by lfarah** [comment]
@RuiAAPeres changed!

**Comment by RuiAAPeres** [comment]
After thinking about this, it doesn't really make sense. Even a developer with very little experience, would know this.

**Comment by lfarah** [comment]
@RuiAAPeres I respect your decision, but [here's another person who didn't know how to manually install](https://github.com/xhzengAIB/PopMenu/issues/15)

**Comment by lfarah** [comment]
@RuiAAPeres here's another example, now from [MailOnline's own library, ImageViewer](https://github.com/MailOnline/ImageViewer/issues/36)

**Pull request closed without merging**